test/tools/afl_fuzzer.cpp

```diff
@@ -93,18 +97,47 @@ Allowed options)",
 		return 0;
 	}
 
-	string input;
+	vector<string> inputs;
 	if (arguments.count("input-file"))
-		input = readFileAsString(arguments["input-file"].as<string>());
+		inputs.push_back(arguments["input-file"].as<string>());
+	else if (arguments.count("input-files"))
+		inputs = arguments["input-files"].as<vector<string>>();
 	else
-		input = readStandardInput();
+		inputs.push_back("");
 
-	if (arguments.count("const-opt"))
-		FuzzerUtil::testConstantOptimizer(input, quiet);
-	else if (arguments.count("standard-json"))
-		FuzzerUtil::testStandardCompiler(input, quiet);
-	else
-		FuzzerUtil::testCompiler(input, !arguments.count("without-optimizer"), quiet);
+	bool optimize = !arguments.count("without-optimizer");
+	int retResult = 0;
+
+	for (string const& inputFile: inputs)
+	{
+		string input;
+		if (inputFile.size() == 0)
```

Comment:
It is a little awkward to see that inside the loop here. Is this actually reached?

Reply:
only if the file name was passed via std in